src/store/run-process-panel/run-process-panel-actions.ts

```diff
@@ -55,10 +56,11 @@ export const runProcess = async (dispatch: Dispatch<any>, getState: () => RootSt
     const basicForm = getFormValues(RUN_PROCESS_BASIC_FORM)(state) as RunProcessBasicFormData;
     const inputsForm = getFormValues(RUN_PROCESS_INPUTS_FORM)(state) as WorkflowInputsData;
     const advancedForm = getFormValues(RUN_PROCESS_ADVANCED_FORM)(state) as RunProcessAdvancedFormData;
+    const userUuid = getState().auth.user!.uuid;
     const { processOwnerUuid, selectedWorkflow } = state.runProcessPanel;
     if (selectedWorkflow) {
         const newProcessData = {
-            ownerUuid: processOwnerUuid,
+            ownerUuid: isItemNotInProject || !isProjectRoute ? userUuid : processOwnerUuid,
```

Comment:
You could create a method whitch return `boolean` from this condition `isItemNotInProject || !isProjectRoute`.